auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -131,7 +132,9 @@ AuthResult validateToken(@PathVariable("authtoken") String token) {
                 //check for valid uuid
                 UUID.fromString(token);
                 authResult = (AuthResult) SessionHolder.getAuthResult(token);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00122)